docs/source/conf.py

```diff
@@ -33,6 +36,7 @@
     "sphinx_rtd_theme",
     "sphinx.ext.intersphinx",
     "pydoctor.sphinx_ext._help_output",
+    "pydoctor.sphinx_ext.api_output",
```

Comment:
Thanks a lot, this is a game changer for Sphinx users to use PyDoctor!

May be the Sphinx plugin could be named "pydoctor.sphinx_ext.build_docs" or  "pydoctor.sphinx_ext.build_api_docs"?

Just to emphasis that the goal of the thing is to build API references as part of the Sphinx build.

Reply:
maybe `build_apidocs` ?

I agree that `api_output` is a stupid name. Thanks for the feedback.

Reply:
> Thanks a lot, this is a game changer for Sphinx users to use PyDoctor!

Is this usable for other projects? I was under the impression that, at least right now, it only works for pydoctor's own documentation.

Reply:
> Is this usable for other projects?

Yes I think so. 

[This is a documentation draft]
As I understand it's pretty straight  forward to integrate pydoctor's build into sphinx with this:

- enable `pydoctor.sphinx_ext.build_apidocs` in `conf.py`
- configure `pydoctor_args`, `pydoctor_cwd`
- insert a `api/index.rst` file with a title and reference it from the Sphinx TOC tree

That's it, the API reference should be under the `api` folder when running your sphinx build!